src/components/FormComponent.jsx

```diff
@@ -1,60 +1,39 @@
-import React from 'react'
-import { Field, formValueSelector, reduxForm } from 'redux-form'
-import { connect } from 'react-redux'
-import countryList from './countries'
-import Input from './input'
-import Select from './select'
-import { maxLength, name, required, tinorssn } from './validators'
+import React, { Component, PropTypes } from 'react'
```

Comment:
change the import for PropTypes from the prop-types library
for reference use:
`import PropTypes from 'prop-types'`

Reply:
Yes it worked. Thanks!